session_security/static/session_security/script.js

```diff
@@ -66,7 +66,12 @@ yourlabs.SessionSecurity.prototype = {
 
     // Called by click, scroll, mousemove, keyup.
     activity: function() {
-        this.lastActivity = new Date();
+        var now = new Date();
+        if (now - this.lastActivity < 1000)
```

Comment:
Could this be maybe an instance setting ?

Could we maybe default it to maybe 100 instead of 1000 ? 1 second would give a feeling of lag maybe but I haven't tested it yet.